docs/content/en/main/concepts/syncer.md

```diff
@@ -96,14 +96,40 @@ You can then create foo.bar in this workspace, or create an APIBinding in anothe
 To sync resource from another existing APIExport in the KCP server, run
 
 ```sh
-kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace|another-apiexport -o syncer.yaml
+kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace:another-apiexport -o syncer.yaml
 ```
 
 Syncer will start syncing the resources in this APIExport as long as the physical clusters has the compatible API schemas.
 
 To see if a certain resource is supported to be synced by the syncer, you can check the state of the `syncedResources` in `SyncTarget`
 status.
 
+### Bind workspace to the location workspace
```

Comment:
```suggestion
### Bind workspaces to the location workspace
```

Reply:
```suggestion
### Bind `kcp` Workspaces to the Location Workspace
```

(use title case)

Reply:
@stevekuznetsov Why **`kcp` Workspaces** ? **Location Workspaces** are also `kcp` workspaces aren't they ? Do you mean **User Workspaces** ?

Reply:
I don't think using `kcp` is useful here. There's not other concept of `workspaces` apart from KCP workspaces.

I agree that "Location Workspace" is something special, and we could title-case it to make it clearer. Then you can bind any workspace (nothing special) to this "Location Workspace".
Since the `location workspace` has been title-cased, it seems sufficient to me. Let me resolve this.